contracts/LifToken.sol

```diff
@@ -12,4 +12,54 @@ contract LifToken is MintableToken, Pausable {
 
   // Token decimals
   uint public constant DECIMALS = 18;
+
+  // Edit of the ERC20 token events to support data argument
```

Comment:
Instead of "Edit of the ERC20 token events...", I think it should be something like "Addition on top of ERC20 token events..." or "Variation of ERC20 token events ..."